framework/src/Volo.Abp.AspNetCore/Volo/Abp/AspNetCore/Auditing/AbpAuditingMiddleware.cs

```diff
@@ -94,8 +94,23 @@ public async Task InvokeAsync(HttpContext context, RequestDelegate next)
 
     private bool IsIgnoredUrl(HttpContext context)
     {
-        return context.Request.Path.Value != null &&
-               AspNetCoreAuditingOptions.IgnoredUrls.Any(x => context.Request.Path.Value.StartsWith(x));
+        if (context.Request.Path.Value == null)
+        {
+            return false;
+        }
+        
+        if (!AuditingOptions.IsEnabledForIntegrationServices && 
+            context.Request.Path.Value.StartsWith($"/{AbpAspNetCoreConsts.DefaultIntegrationServiceApiPrefix}/"))
```

Comment:
this maybe a problem, because the router prefix of `IntegrationServiceApi` is configureable

Reply:
I know and you are right.
If we set another prefix while configuring conventional controllers for application services, then we should also add that URL to the `AbpAspNetCoreAuditingOptions.IgnoredUrls`. ABP can't make it automatic, because a conventional controller configuration may contain both regular application and integration services those share a single `ApiRoutePrefix`.
I suppose it is best to remove the `ApiRoutePrefix` option since it may break other things. Thanks for your comment.